README.md

```diff
@@ -66,10 +69,15 @@ OPML Import
 Selfoss supports importing OPML files. Find the OPML export in the old application, it is usually located somewhere in settings. Then visit the page https://yourselfossurl.com/opml and upload it there.
 
 
-APPS
+MOBILE APPS
```

Comment:
Maybe this could be called third-party apps, since I want to add support to https://github.com/jangernert/FeedReader once we stabilize the API for 3.0.

Reply:
Though, I am not sure if we want to include this section in the README. I am still struggling with what should go where: README, website or both.